utils/imagenet_topk_accuracy_driver.py

```diff
@@ -0,0 +1,318 @@
+                    help="Command to use for running the image-classifier, "
+                    "including the binary and all of its command lime "
+                    "parameters.")
+
+def pil_loader(path):
+    # open path as file to avoid ResourceWarning
+    # (https://github.com/python-pillow/Pillow/issues/835)
+    with open(path, 'rb') as f:
+        img = Image.open(f)
+        return img.convert('RGB')
+
+def accimage_loader(path):
+    import accimage
+    try:
+        return accimage.Image(path)
+    except IOError:
+        # Potentially a decoding problem, fall back to PIL.Image
+        return pil_loader(path)
+
+def default_image_loader(path):
+    from torchvision import get_image_backend
+    if get_image_backend() == 'accimage':
+        return accimage_loader(path)
+    else:
+        return pil_loader(path)
+
+
+## @returns two lists of the same length found in directory @param
+# validation_images_dir; the first list contains paths to all images found, and
+# the second list contains the corresponding labels of the image.
+def get_img_paths_and_labels(validation_images_dir):
+    assert os.path.exists(validation_images_dir), "Validation directory did not exist: " + validation_images_dir
+
+    # Get and sort the subdirectories containing images.
+    img_dir_paths = []
+    for d in os.listdir(validation_images_dir):
+        dir_path = os.path.join(validation_images_dir, d)
+        if os.path.isdir(dir_path):
+            img_dir_paths.append(dir_path)
+    img_dir_paths.sort()
+
+    # Create lists holding paths to each image to be classified and the label
+    # for that image.
+    img_paths = []
+    img_labels = []
+    curr_label_idx = 0
+    for img_dir_path in img_dir_paths:
+        for d in os.listdir(img_dir_path):
+            full_img_path = os.path.join(img_dir_path, d)
+            if os.path.isfile(full_img_path):
+                img_paths.append(full_img_path)
+                img_labels.append(curr_label_idx)
+        curr_label_idx = curr_label_idx + 1
+    return img_paths, img_labels
+
+## Given an image located at @param img_path, transform the image
+# and save it to the path @param path_to_new_img.
+def resize_and_save_image(img_path, path_to_new_img):
+    # Load the image.
+    img = default_image_loader(img_path)
+
+    # Use to Resize and CenterCrop the images to 224x224.
+    transform_resize = transforms.Compose([
+        transforms.Resize(256),
+        transforms.CenterCrop(224),
+    ])
+
+    resized_img = transform_resize(img)
+    resized_img.save(path_to_new_img, format="png")
+
+## Used to pre-process an input set of images. Takes a string of a directory
+# @param validation_images_dir and saves the cropped subset of the images in a
+# subdirectory `processed/`, which must not yet exist.
+def save_centered_cropped_dataset(validation_images_dir):
+    assert os.path.exists(validation_images_dir), "Validation directory does not exist: " + validation_images_dir
```

Comment:
This and some other lines are longer than 80 chars. Could you use pylint or something like this?